code/modules/shuttle/special.dm

```diff
@@ -255,10 +255,14 @@
 	use_power = NO_POWER_USE
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 	speech_span = SPAN_ROBOT
+	///Cumulative amount of credits required to pass
 	var/threshold = 500
+	///Amount of credits paid so far, will let you pass once threshold is met
+	var/list/payees = list()
+	///Stores the amount of cash or credits paid towards the scanner
+	var/list/counted_money = list()
```

Comment:
What does this do that payees can't?

Reply:
I honestly dont know, I was half tempted to just delete it but it's old code that I'm unsure how it works exactly.

Reply:
If it doesn't do anything, remove it. 
If it does do anything, see if it's replaceable by ``payees`` instead